obs-studio-client/source/input.hpp

```diff
@@ -26,6 +26,14 @@ namespace osn
 {
 	class Input : public osn::ISource, public utilv8::ManagedObject<osn::Input>
 	{
+		enum FilterMovement
```

Comment:
int values of this enums have fixed values on frontend. 
So better to have them explicitly set in this enum too and not expect that compiler always generate them exactly as we need.

Reply:
Ok! Making the change